app/controllers/openid_connect/logout_controller.rb

```diff
@@ -17,6 +24,19 @@ def index
         success: result.success?,
       )
 
+      if result.success? && (redirect_uri = result.extra[:redirect_uri])
+        handle_successful_logout_request(redirect_uri)
+      else
+        render :error
+      end
+    end
+
+    def delete
+      @logout_form = OpenidConnectLogoutForm.new(
```

Comment:
Could be good to refactor a method here?

```rb
def logout_form
  OpenidConnectLogoutForm.new(
    params: logout_params,
    current_user: current_user,
  )
end
```

Or:

```rb
before_action :set_logout_form

attr_reader :logout_form

def set_logout_form
  @logout_form = OpenidConnectLogoutForm.new(
    params: logout_params,
    current_user: current_user,
  )
end
```

Reply:
refactored in eba6226